coap-gateway/service/observation/deviceObserver.go

```diff
@@ -373,7 +373,7 @@ func createDiscoveryResourceObserver(ctx context.Context, deviceID string, coapC
 // Create observer with a single observations for all published resources.
 func createPublishedResourcesObserver(ctx context.Context, deviceID string, coapConn ClientConn, callbacks ResourcesObserverCallbacks, published []*commands.Resource, logger log.Logger) (*resourcesObserver, error) {
 	resourcesObserver := newResourcesObserver(deviceID, coapConn, callbacks, logger)
-
+	// TODO get ETAG for each resource
```

Comment:
Is this waiting for some not yet done IoT-lite code?

Reply:
Observation with ETAGs :)

Reply:
Please mentioned in in the documentation and config of helm. That in case batch observe is not supported, etags won't work.